spec/dns_spec.rb

```diff
@@ -0,0 +1,118 @@
+describe LinuxAdmin::Dns do
+  RESOLV_CONF = <<DNS_END
+# Generated by NetworkManager
+search test.one.com test.two.com. test.com.
```

Comment:
Please don't use public domains and addresses.  Prefer those reserved for testing and documentation.
Domains: `example.com` and `example.org` and sub domains.
IP addresses: `10.*.*.*`, `172.16.*.* - 172.18.*.*` and `192.168.*.*`